cdci_data_analysis/analysis/renku_helper.py

```diff
@@ -287,15 +291,20 @@ def create_renku_ini_config_obj(repo, default_url_file_name):
 
     renku_ini_path = os.path.join(repo_dir, '.renku', 'renku.ini')
 
-    renku_config = ConfigParser()
-    renku_config.read(renku_ini_path)
+    config_ini_obj = ConfigParser()
+    config_ini_obj.read(renku_ini_path)
 
     try:
-        renku_config['renku "interactive"']['default_url'] = f'/lab/tree/{default_url_file_name}'
-    except KeyError:
-        renku_config['interactive']['default_url'] = f'/lab/tree/{default_url_file_name}'
+        if 'renku "interactive"' in config_ini_obj:
```

Comment:
but you are still keeping the old-style key, why? We want to discard them

Reply:
Because I don't create any new file, I just change the content by modifying the `default_url` entry, and I made it in a way that it still supports the old approach without modifying its content.

Reply:
You are "modifying the default_url entry" and later write it in a file, right? How is this "without modifying its content"?

Reply:
You do not just modify, you actually write a file from scratch from the ini object which you constructed from the previous file. You can as well remove the old-style field and add the new-style one. And show a warning that the repository is updated.

Reply:
By the way, how does renku treat this field? Could you have a look in renku-python code to see if the old value is still supported? If no - it's imperative to update to new. If still supported - still good to update, why not? Any disadvantage of upgrading this feature of the repository in this push branch?

Reply:
Just to make sure, did you manage to have a look on this question? @burnout87